WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -122,6 +122,12 @@ public async Task<bool> StartRoundAsync(RoundState roundState, CancellationToken
 
 			var finalRoundState = await RoundStatusUpdater.CreateRoundAwaiter(s => s.Id == roundState.Id && s.Phase == Phase.Ended, cancellationToken).ConfigureAwait(false);
 
+			if (finalRoundState.CoinjoinState is SigningState state && state.IsFullySigned)
+			{
+				// TODO cancellation token? broadcast ourselves?
```

Comment:
Currently we broadcast what the backend already did. After that, there is a rebroadcast mechanism in the clients but only after. Now that we have the full CJ in our hand at this point, I think we should broadcast it. My concept is that if the tx is signed and valid, then anyone can do it, so we should be on the same page.

Reply:
cancellationToken is fine as we must have a limit for the whole operation. After the timeout kicks in we can broadcast!?